### DISCUSSION TITLE
How can I join to learn the project and support?

### DISCUSSION THREAD

**Discussion by navisoft** [zephir-lang/zephir#2250] [General]
I have some projects written by the Zephir, but I heard that the team will support it anymore. So I want to join the project to learn and can continue use the Zephir for my project in the future. Please let me know how can I join the project.

**Comment by Jeckerson** [comment]
Hello

Zephir project will continue to exist even after Phalcon will migrate to PHP native, as Phalcon will still continue to have core as extension in Zephir + C.

If you want really help, you can start to pick issues with recent milestones and submit valid PRs (with tests):
https://github.com/zephir-lang/zephir/issues?q=is%3Aopen+is%3Aissue+milestone%3A0.13.x

Also, we are all located in our [Discord server](https://phalcon.io/discord) on daily basis, so if you want to discuss something related to Zephir - welcome.

**Reply by navisoft** [reply]
Hi @Jeckerson 
Thank you for the feedback, currently, I'm still not confident with the C enough and not really understand the project as well. I need to learn more before can start to do something.
Thank you!
